cmd/expiration-mailer/main.go

```diff
@@ -484,6 +488,17 @@ func main() {
 	cmd.FailOnError(err, "Failed to load credentials and create gRPC connection to SA")
 	sac := bgrpc.NewStorageAuthorityClient(sapb.NewStorageAuthorityClient(conn))
 
+	var smtpRoots *x509.CertPool
+	if c.Mailer.SMTPTrustedRootFile != "" {
+		pem, err := ioutil.ReadFile(c.Mailer.SMTPTrustedRootFile)
+		cmd.FailOnError(err, "Loading trusted roots file")
+		smtpRoots = x509.NewCertPool()
+		ok := smtpRoots.AppendCertsFromPEM(pem)
+		if !ok {
```

Comment:
I think you can collapse this into the assignment above: `if !smtpRoots.AppendCertsFromPEM(pem) {`